pydantic/errors.py

```diff
@@ -113,10 +113,14 @@ class PathNotADirectoryError(_PathValueError):
     msg_template = 'path "{path}" does not point to a directory'
 
 
-class PyObjectError(PydanticTypeError):
+class PyObjectImportError(PydanticTypeError):
```

Comment:
I'm worried the change in error `type` could mess up some people's interfaces. Please set `type` explicitly to ensure it remains unchanged.

Reply:
changed the error name back to what it was, fixed the name for the new error type